### ISSUE TITLE
[deeplab] How do you preprocess image when training xception_65 in imagenet?

### ISSUE THREAD

**Issue by KeyKy** [tensorflow/models#4324]
How to reproduce the accuracy of xception_65 in imagenet? Thanks!

**Comment by aquariusjay** [comment]
We use [slim](https://github.com/tensorflow/models/tree/master/research/slim) to pretrain xception_65 on imagenet.
We basically use the same setting as how slim trains ResNet model variants except that we use ['inception_preprocessing.py'](https://github.com/tensorflow/models/blob/master/research/slim/preprocessing/inception_preprocessing.py).

**Comment by jianlong-yuan** [comment]
@KeyKy Have you reproduce the result?
